### PR TITLE
Feat/index and products

### PR DESCRIPTION
**Pull request by tonymike99**
I've added the index and products page.

image
image

### CODE REVIEW

**Review by bossmonkey98 — APPROVED** [review]

File: assets/css/index.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        'main main main main main'
+        'main main main main main'
+        'footer footer footer footer footer';
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: 0.3fr repeat(3, 1fr) 0.6fr;
+}
+
+/* COMMON STYLES MODIFICATION END */
+
+/* **************************************************************************************************** */
```
Comment (iRohitGaur) [inline assets/css/index.css:23]: nice commented code. easy to understand

File: assets/css/index.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+/* COMMON STYLES MODIFICATION START */
+
+.main-container {
+    grid-template-areas:
+        'header header header header header'
+        'main main main main main'
+        'main main main main main'
+        'main main main main main'
+        'footer footer footer footer footer';
```
Comment (iRohitGaur) [inline assets/css/index.css:16]: you can do this here:

grid-template-areas:
        'header'
        'main'
        'footer';

File: assets/css/styles-common.css [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    --secondary-hover: #5c636a;
+    --success-hover: #157347;
+    --danger-hover: #bb2d3b;
+    --warning-hover: #ffca2c;
+    --info-hover: #31d2f2;
+    --light-hover: #f9fafb;
+    --dark-hover: #1c1f23;
+
+    --link-hover-bg: #36096d;
+    --default-bg: yellowgreen;
```
Comment (iRohitGaur) [inline assets/css/styles-common.css:36]: use hex for consistency

File: assets/css/styles-common.css [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+.header {
+    grid-area: header;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 1rem 6rem;
+    background-color: var(--dark-color);
+    color: var(--light-color);
+    /* border-bottom: 2px solid black; */
```
Comment (iRohitGaur) [inline assets/css/styles-common.css:80]: remove commented code

File: assets/css/styles-common.css [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+    text-decoration: none;
+}
+
+iframe {
+    height: 100% !important;
+    margin-bottom: 5rem;
+    max-height: 20rem;
+    overflow: scroll;
+    overflow-x: scroll;
+    width: 100% !important;
```
Comment (iRohitGaur) [inline assets/css/styles-common.css:234]: don't use !important. Try overiding it with specificity.

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes

**Pull request merged**